"Shoot constantly when active" causes build errors

### Issue description

When creating a ranged weapon, if "Shoot constantly when active" is checked. The mod will not compile.

### How to reproduce this issue?

Step 1. Make a ranged weapon and configure it any way you want.
Step 2. Check the "Shoot constantly when active" box.
Step 3. Save.

### Operating system

Windows

### Details

MCreator 2022.3
Latest Development Build (what i mean is that i got this version from downloading the source and zipping the "src" folder.)

### Example workspace

[shootconstantlytest.zip](https://github.com/Goldorion/Fabric-Generator-MCreator/files/10097596/shootconstantlytest.zip)


### Logs

```
Executing Gradle task: build 
Build info: MCreator 2022.3.48217, fabric-1.19.2, 64-bit, 16332 MB, Windows 10, JVM 17.0.3, JAVA_HOME: C:\Users\Micasddsa\Downloads\MCreator20223\jdk, started on: 2022-11-26-21:24:58 
  
> Configure project : 
Fabric Loom: 1.0.12 
Not publishing sources jar as it was not found. Use java.withSourcesJar() to fix. 
> Task :compileJava FAILED 
[C:\Users\Micasddsa\MCreatorWorkspaces\shootconstantlytest\src\main\java\net\mcreator\shootconstantlytest\item\RangedWeaponItem.java:42](file:///C:/Users/Micasddsa/MCreatorWorkspaces/shootconstantlytest/src/main/java/net/mcreator/shootconstantlytest/item/RangedWeaponItem.java:42): error: method does not override or implement a method from a supertype 
   @Override 
   ^ 
1 error 
FAILURE: Build failed with an exception. 
* What went wrong: 
Execution failed for task ':compileJava'. 
> Compilation failed; see the compiler error output for details. 
* Try: 
> Run with --stacktrace option to get the stack trace. 
> Run with --info or --debug option to get more log output. 
> Run with --scan to get full insights. 
* Get more help at https://help.gradle.org 
BUILD FAILED in 2s 
1 actionable task: 1 executed 
```

### Issue tracker rule checks (please read carefully)

- [X] I have checked that my problem <a href='https://github.com/MCreator/MCreator/issues?q=is%3Aissue' target='_blank'>is not already reported</a>
- [X] I have checked that my problem is not covered on <a href='https://mcreator.net/support/knowledgebase' target='_blank'>Knowledge Base</a> or on <a href='https://mcreator.net/wiki' target='_blank'>MCreator's Wiki</a>
- [X] I have checked that my written content does not violate the <a href='https://mcreator.net/wiki/general-publishing-guidelines' target='_blank'>publishing guidelines</a>

--- a/src/fabric-1.19.2/templates/rangeditem/rangeditem.java.ftl
+++ b/src/fabric-1.19.2/templates/rangeditem/rangeditem.java.ftl
@@ -90,7 +90,7 @@ public class ${name}Item extends Item {
 
 	<#if data.shootConstantly>
 		@Override
-		public void onUseTick(Level level, LivingEntity livingEntity, ItemStack itemstack, int i) {
+		public void onUseTick(Level level, LivingEntity livingEntity, ItemStack itemstack, int count) {
 			Level world = livingEntity.level;
 			if (!world.isClientSide() && livingEntity instanceof ServerPlayer entity) {
 				double x = entity.getX();

--- a/src/fabric-1.19.2/templates/rangeditem/rangeditem.java.ftl
+++ b/src/fabric-1.19.2/templates/rangeditem/rangeditem.java.ftl
@@ -89,9 +89,10 @@ public class ${name}Item extends Item {
 	</#if>
 
 	<#if data.shootConstantly>
-		@Override public void onUsingTick(ItemStack itemstack, LivingEntity entityLiving, int count) {
-			Level world = entityLiving.level;
-			if (!world.isClientSide() && entityLiving instanceof ServerPlayer entity) {
+		@Override
+		public void onUseTick(Level level, LivingEntity livingEntity, ItemStack itemstack, int i) {
+			Level world = livingEntity.level;
+			if (!world.isClientSide() && livingEntity instanceof ServerPlayer entity) {
 				double x = entity.getX();
 				double y = entity.getY();
 				double z = entity.getZ();

